src/Reflection/Adapter/ReflectionType.php

```diff
@@ -7,20 +7,19 @@
 use ReflectionType as CoreReflectionType;
 use Roave\BetterReflection\Reflection\ReflectionIntersectionType as BetterReflectionIntersectionType;
 use Roave\BetterReflection\Reflection\ReflectionNamedType as BetterReflectionNamedType;
+use Roave\BetterReflection\Reflection\ReflectionType as BetterReflectionType;
 use Roave\BetterReflection\Reflection\ReflectionUnionType as BetterReflectionUnionType;
 
 use function array_filter;
 use function array_values;
+use function assert;
 use function count;
 
 abstract class ReflectionType extends CoreReflectionType
 {
-    public static function fromTypeOrNull(BetterReflectionNamedType|BetterReflectionUnionType|BetterReflectionIntersectionType|null $betterReflectionType): ReflectionUnionType|ReflectionNamedType|ReflectionIntersectionType|null
+    /** @internal */
+    public static function fromType(BetterReflectionNamedType|BetterReflectionUnionType|BetterReflectionIntersectionType|null $betterReflectionType): ReflectionUnionType|ReflectionNamedType|ReflectionIntersectionType
```

Comment:
How do we kill this mutant, before merging? 🤔

Reply:
Added tests but the mutant is still there 🤷‍♂️

Reply:
Let's roll with the current solution for now :+1: